### ISSUE TITLE
RuntimeError: sys.stderr is None

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#2710]
https://sentry.io/quodlibet/quodlibet/issues/432901998/

```
RuntimeError: sys.stderr is None
  File "<string>", line 1, in <module>
  File "quodlibet/main.py", line 197, in main
    quodlibet.run(window, before_quit=before_quit)
  File "quodlibet/_main.py", line 338, in run
    faulthandling.enable(os.path.join(get_user_dir(), "faultdump"))
  File "quodlibet/errorreport/faulthandling.py", line 84, in enable
    faulthandler.enable(_fileobj, all_threads=False)

RuntimeError: sys.stderr is None
```

```
After moving the "portable" version I get such errors. martin.lemburg@gmx.net
```

**Comment by MartinL0815** [comment]
Here I am.

About the the way I get to the problem:

1. I unzipped quotlibet in a temporary directory to try out
2. I configured some plugins, added two library folders
3. I switched through some "views"
4. I opened the SoundCloud view and clicked "connect"
5. I closed quotlibet
6. Moved quotlibet to the root directory of my portable applications
7. I opened quotlibet again and …
    - saw initially the SoundCloud view
    - missed some images in the UI (play, pause, shuffle, etc.)
    - switched to the cover art view
    - got error messages containing the old folders
    - exited the process
8. Started the process again and got the error above, which is raised always after startup
9. Ignoring the error lets quotlibet become "not responding".

**Comment by MartinL0815** [comment]
Sorry, forgot about providing the operating system: MS Windows 7 SP2 Enterprise Edtion 64bit

**Comment by lazka** [comment]
Thanks, I'll try to reproduce.

**Comment by MartinL0815** [comment]
I tried to replace the current runtime environment (data directory) by starting the 7z executable and extract to the desired directory, which ended in the following:

> 
![extract errors from 7z executable](https://user-images.githubusercontent.com/525700/34879023-2407bb4a-f7ac-11e7-857b-658b4da233d2.png)

Then I used the 7z installation to extract the files from the explorer context menu to the desired directory:

> 
![successful manual extraction](https://user-images.githubusercontent.com/525700/34879059-4aa31cd6-f7ac-11e7-96a5-1852c174b580.png)

I replaced the complete distribution and restarted the "setup" by switching the theme to a "dark" high-contrast-theme first to continue with the rest.

After having listened to some tracks and letting quodlibet scan a part of my big library, I exited quodlibet and restarted it without a problem.
So I was not able to reproduce the issue in this simple way.
